doc/releases/changelog-dev.md

```diff
@@ -121,6 +120,10 @@
 
 <h3>Improvements 🛠</h3>
 
+* A more resource efficient decomposition rule has been added to :class:`~.QubitUnitary` that works for 
+  any number of qubits.
+  [(#7277)](https://github.com/PennyLaneAI/pennylane/pull/7277)
```

Comment:
Can we show what the decomposition rule is, maybe? You can print the decomposition rule with print(qml.list_decomps(qml.QubitUnitary)[index]) and it will show the quantum function + resources registered :)

Reply:
Oh so this only works with the new system disabled?

Reply:
Ya Issac, here Guillermo improved the compute_decomposition method instead of adding a decomposition rule and registering resources. I think we should change the wording here so that we don't say "decomposition rule" as this could cause confusion with other developers.